### DISCUSSION TITLE
what is dt mean in the coed

### DISCUSSION THREAD

**Discussion by neilgogogo** [NVlabs/instant-ngp#315] [General]
In the code, I saw many dt in the code, I read the code but I have no idea what is dt correspond? It seems that dt and pos could determine the mip.

Many thanks

https://github.com/NVlabs/instant-ngp/blob/7b37eee0aa9ea31362d2a5a799e007a731de385c/src/testbed_nerf.cu#L628

**Comment by Tom94** [comment]
It's the step size for ray marching.

**Comment by JoanCharmant** [comment]
It's `dt` as in "derivative of t", `t` being the typical variable name used for the distance along a ray. `position = origin + (t * direction)`. So `dt` is the displacement per step.
